src/MonologCreator/Factory.php

```diff
@@ -130,6 +130,13 @@ public function createProcessors(array $loggerConfig)
                 $processors[] = $webProcessor;
             } elseif ('requestId' === $processor) {
                 $processors[] = new Processor\RequestId();
+            } elseif ('extraField') {
+                $extraFields = null;
+                if (is_array($loggerConfig['extraFields'])) {
```

Comment:
Please add an empty line above this `if` block.

Please make it an explicit boolean compare as in `if (true === is_array(...))`